feature-libs/tms/tsconfig.lib.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/lib",
+    "target": "es2015",
+    "module": "es2020",
+    "moduleResolution": "node",
+    "declaration": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "types": [],
+    "lib": ["dom", "esnext"],
+    "paths": {
+      "@spartacus/core": ["dist/core"]
+    }
+  },
+  "angularCompilerOptions": {
+    "skipTemplateCodegen": true,
```

Comment:
What about adding `strictTemplates` in new libs, as it's the most strict AFAIK? 
(I know, it won't change here much, but as a convention).